src/App.css

```diff
@@ -0,0 +1,301 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,700;0,900;1,100;1,300&family=Quicksand:wght@300;400;500;600;700&family=Saira:ital,wght@0,100;0,200;0,400;0,500;0,600;1,100;1,300&display=swap');
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  font-family: 'Quicksand', sans-serif;
+}
+
+/*** NAVBAR ***/
+
+body {
+  background-color: aliceblue;
+  padding: 1rem;
+}
+
+nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 10px;
+  border-bottom: 1px solid rgb(72, 72, 72);
+}
+
+/* .left {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+} */
```

Comment:
- [ ] Kindly make sure to remove all commented code in order to always keep your codebase clean and professional. :wink: